database/seed.js

```diff
@@ -0,0 +1,55 @@
+const mysql = require('mysql');
+const faker = require('faker');
+const moment = require('moment');
+
+// Open a connection to the database
+const connection = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "password!",
+  database: "fec_reviews"
+});
+
+// Populate reviews table
+connection.connect((err) => {
+  if (err) throw err;
+  console.log('Database connected to populate reviews table!');
+
+  let reviews_count = 0;
+  let images_count = 0;
+
+  while (reviews_count < 100) {
+    // Use faker to create test data
+    var username_fake = faker.name.firstName();
+    var fecha_fake = faker.date.past();
+    var formattedDate = JSON.stringify(fecha_fake).split('T')[0].split('"').join('')
+    var rating_fake = Math.floor(Math.random() * (5 - 1) + 1);
+    var title_fake = faker.random.words();
+    var review_body_fake = faker.lorem.paragraph();
```

Comment:
I like the way you did this! I dig the way you put formattedDate together: very terse.